### PR TITLE
acceptance/cluster: `Consistent` returns error

### PR DESCRIPTION
**Pull request by tamird**
Intended to help debug #12015.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12183)

### CODE REVIEW

File: pkg/acceptance/chaos_test.go [hunk 1]
```diff
@@ -254,7 +254,9 @@ func chaosMonkey(
 			time.Sleep(time.Second)
 		}
 		c.Assert(ctx, state.t)
-		cluster.Consistent(ctx, state.t, c)
+		if err := cluster.Consistent(ctx, c); err != nil {
+			state.t.Error(err)
```
Comment (tamird) [inline pkg/acceptance/chaos_test.go]: Note that this differs from the old behaviour which called `Fatal`, but this change is necessary due to the unsafety of calling `Fatal` in the non-main goroutine now being detected by staticcheck.

I've filed an issue upstream: https://github.com/dominikh/go-staticcheck/issues/60 to track the naivete if staticcheck in this regard.

**Review by a-robinson — APPROVED** [review]
This helps with debugging because now the chaos thread won't crash the main test thread, I take it?

**Comment by a-robinson** [comment]
For what it's worth, the test failed in the teamcity run on this PR

**Comment by tamird** [comment]
Not, it helps with debugging because now the reported error has a sensible line number. In fact, this PR failed CI on that issue. https://teamcity.cockroachdb.com/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=84594#_focus=379:
```
[18:22:17][TestNodeRestart] 	chaos_test.go:258: failed to send RPC: rpc error: code = 14 desc = grpc: the connection is unavailable
[18:22:17][TestNodeRestart] 	chaos_test.go:258: failed to send RPC: rpc error: code = 14 desc = grpc: the connection is unavailable
```

**Comment by a-robinson** [comment]
The 2nd commit LGTM as well.

**Pull request merged**